index.js

```diff
@@ -37,12 +37,11 @@ function sitedown (options, callback) {
       return entry.path
     }))
     .on('data', function (file) {
-      var dest = file.split('/').slice(0, -1).join('/')
-      var name = file.split('.')[0].split('/').pop()
+      var parsedFile = path.parse(file)
+      if (parsedFile.name === 'README') parsedFile.name = 'index'
+      parsedFile.base = parsedFile.name + '.html'
 
-      if (name === 'README') name = 'index'
-
-      dest = path.join(dest, name + '.html')
+      var dest = path.format(parsedFile)
```

Comment:
Ah shoot. I was going to send this separately.

This switches to leverage `path` instead to avoid possible Windows incompatibilities (`/` vs `/`).